### PR TITLE
Added Flutter PDF viewer Sticky note feature content

### PR DESCRIPTION
**Pull request by immankumarsync**
Added Flutter PDF viewer Sticky note feature content

### CODE REVIEW

File: Flutter/pdf-viewer/sticky-notes.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+title: Sticky Note Annotations in Flutter PDF Viewer Widget | Syncfusion
+description: Learn here all about sticky note annotations in the Syncfusion Flutter PDF Viewer (SfPdfViewer) widget and more.
+platform: flutter
+control: SfPdfViewer
+documentation: ug
+---
+
+# Sticky Note Annotations in Flutter PDF Viewer Widget (Syncfusion)
+
+The sticky note annotation feature of `SfPdfViewer` allows you to add, remove, and modify sticky notes in a PDF document. This feature will help add comments or notes to specific parts of a document to clarify complex concepts, terms, or ideas. This section will cover the various functions available in `SfPdfViewer` for working with sticky note annotations.
```
Comment (Deepak1799) [inline Flutter/pdf-viewer/sticky-notes.md:12]: Can this be API linked? 
image

File: flutter-toc.html [hunk 2]
```diff
@@ -494,6 +494,7 @@
                     <li>  Types
```
Comment (Deepak1799) [inline flutter-toc.html:494]: * Overview page needs to updated? https://help.syncfusion.com/flutter/pdf-viewer/overview
* Similarly, localization, and other common things.

File: Flutter/pdf-viewer/sticky-notes.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
+---
```
Comment (Deepak1799) [inline Flutter/pdf-viewer/sticky-notes.md:1]: Can we provide border or shadow for image?

image

File: Flutter/pdf-viewer/sticky-notes.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+layout: post
```
Comment (Deepak1799) [inline Flutter/pdf-viewer/sticky-notes.md:2]: Can these be API linked?

image

File: Flutter/pdf-viewer/sticky-notes.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+The following sticky note icon types are currently available in `SfPdfViewer`. The icon types can be selected from the [PdfStickyNoteIcon](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/PdfStickyNoteIcon.html) enumeration.
+
+![Sticky note icon types](images/annotations/sticky-note-icon-types.png)
+
+## Add sticky notes
+
+This section will go through how to add sticky note annotations to a PDF document interactively and programmatically.
+
+### Add sticky notes from annotation mode
```
Comment (Deepak1799) [inline Flutter/pdf-viewer/sticky-notes.md:24]: In some places API is linked, and in some other places it was not linked

image

File: Flutter/pdf-viewer/sticky-notes.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Edit the selected sticky note annotation
+
+### Edit the text using UI interaction
+
+When you double-tap the selected sticky note, the text editor opens. Edit the text and click "Save" to modify the text.
```
Comment (Deepak1799) [inline Flutter/pdf-viewer/sticky-notes.md:117]: To opened Editor double tap is used in Flutter also?

image

File: Flutter/pdf-viewer/sticky-notes.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+![Sticky note editing](images/annotations/sticky-notes-edit.gif)
```
Comment (Deepak1799) [inline Flutter/pdf-viewer/sticky-notes.md:119]: image

Here it is mentioned as automatically opens,. but in today's discussion you have siad it was handled from application level. So need to explain based on control level

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

File: Flutter/pdf-viewer/sticky-notes.md (hunk 4 above)
Comment (immankumarsync) [reply Flutter/pdf-viewer/sticky-notes.md:2]: There is no API links is available for the values in enumeration.
https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/PdfAnnotationMode.html

File: Flutter/pdf-viewer/sticky-notes.md (hunk 6 above)
Comment (immankumarsync) [reply Flutter/pdf-viewer/sticky-notes.md:117]: Yes

File: Flutter/pdf-viewer/sticky-notes.md (hunk 7 above)
Comment (immankumarsync) [reply Flutter/pdf-viewer/sticky-notes.md:119]: The edit text dialog is displayed from the source level.

File: Flutter/pdf-viewer/sticky-notes.md (hunk 1 above)
Comment (immankumarsync) [reply Flutter/pdf-viewer/sticky-notes.md:12]: API url linked

File: Flutter/pdf-viewer/sticky-notes.md (hunk 5 above)
Comment (immankumarsync) [reply Flutter/pdf-viewer/sticky-notes.md:24]: API url linked

**Review by Deepak1799 — APPROVED** [review]
Approved

**Pull request merged**